core/src/main/java/com/scalar/db/exception/transaction/ValidationException.java

```diff
@@ -0,0 +1,12 @@
+package com.scalar.db.exception.transaction;
+
+public class ValidationException extends TransactionException {
```

Comment:
@feeblefakie Which do you think is `ValidationException` or `ValidateException` better?

Reply:
As discussed, I won't rename `ValidationException` and will rename `PrepareException` to `PreparationException`.